kolibri/core/assets/src/mixins/commonCoreStrings.js

```diff
@@ -17,6 +17,11 @@ export const coreStrings = createTranslator('CommonCoreStrings', {
     context:
       'Button to cancel an action and return to the previous page. Usually this is the opposite of the save button which saves some piece of information.',
   },
+  learnMoreAction: {
```

Comment:
Also defined in
- MissingresourceAlert (kolibri-common)
- DataPage (facility)
- ProfilePage/index (user_profile)

Could update those in this PR, make a follow-up issue, or just revert this change and add this same string to the GroupsPage component